src/lib.rs

```diff
@@ -6,10 +6,24 @@
 // found in the THIRD-PARTY file.
 
 //! Implements virtio devices, queues, and transport mechanisms.
+extern crate epoll;
 #[macro_use]
 extern crate log;
 extern crate vm_memory;
+extern crate vmm_sys_util;
 
+mod device;
 mod queue;
 
+pub use self::device::*;
 pub use self::queue::*;
+
+#[derive(Debug)]
+pub enum ActivateError {
+    EpollCtl(std::io::Error),
+    BadActivate,
+    #[cfg(feature = "vsock")]
```

Comment:
This doesn't seem relevant at the moment.

Reply:
Fixed, thanks.